src/app/external-cluster-wizard/steps/external-cluster/provider/aks/template.html

```diff
@@ -0,0 +1,171 @@
+             [formControlName]="Controls.Name" />
+      <button mat-icon-button
+              type="button"
+              matSuffix
+              class="km-randomize-btn"
+              matTooltip="Generate name"
+              (click)="generateName()">
+        <i class="km-icon-randomize"></i>
+      </button>
+      <mat-hint>Enter a unique name for this cluster</mat-hint>
+      <mat-error *ngIf="form.get(Controls.Name).hasError(ErrorType.Required)">
+        Name is <strong>required</strong>.
+      </mat-error>
+    </mat-form-field>
+
+    <mat-form-field>
+      <mat-label>Kubernetes Version</mat-label>
+      <input [formControlName]="Controls.KubernetesVersion"
+             matInput
+             type="text"
+             autocomplete="off" />
+
+      <mat-hint>
+        Set the kubernetes version see list of
+        <a href="https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli"
+           rel="noopener noreferrer"
+           target="_blank">supported versions</a>.
+      </mat-hint>
+
+      <mat-error *ngIf="form.get(Controls.KubernetesVersion).hasError(ErrorType.Required)">
+        Version is <strong>required</strong>.
+      </mat-error>
+    </mat-form-field>
+
+    <mat-form-field>
+      <mat-label>Location</mat-label>
+      <input matInput
+             type="text"
+             autocomplete="off"
+             [formControlName]="Controls.Location" />
+      <mat-hint>Enter a region name for your cluster.</mat-hint>
+      <mat-error *ngIf="form.get(Controls.MinCount).hasError(ErrorType.Required)">
+        Location is <strong>required</strong>.
+      </mat-error>
+    </mat-form-field>
+
+    <mat-form-field>
+      <mat-label>Resource group</mat-label>
+      <input matInput
+             type="text"
+             autocomplete="off"
+             [formControlName]="Controls.NodeResourceGroup"
+             required="true" />
+      <mat-hint>Enter name of the resource group containing agent pool nodes.</mat-hint>
+      <mat-error *ngIf="form.get(Controls.MinCount).hasError(ErrorType.Required)">
+        Resource group is <strong>required</strong>.
+      </mat-error>
+    </mat-form-field>
+  </div>
+
+  <div fxFlex="50%"
+       fxLayout="column"
+       fxLayoutGap="8px">
+    <mat-card-header class="km-no-padding">
+      <mat-card-title>Node Pool Setting</mat-card-title>
+    </mat-card-header>
+
+    <km-autocomplete label="VmSize"
+                     required="true"
+                     [isLoading]="isLoadingVmSizes"
+                     [options]="vmSizes"
+                     [formControlName]="Controls.VmSize">
+      <ng-container hint>
+        VM size availability varies by region.
+        <a href="https://docs.microsoft.com/azure/aks/quotas-skus-regions"
+           rel="noopener noreferrer"
+           target="_blank">supported versions</a>.
+      </ng-container>
+    </km-autocomplete>
+
+    <mat-form-field>
+      <mat-label>Node pool name</mat-label>
```

Comment:
```suggestion
      <mat-label>Node Pool Name</mat-label>
```

Update the error message too.